charter.md

```diff
@@ -53,15 +53,164 @@ The following responsibilities belong to the Steering Committe, but may be deleg
 * Control and delegate access to and establish processes regarding
   project resources/assets<sup>[6](#footnote6)</sup>  
 
-## Quorum
+## Changes
 
-Steering committee members are generally expected to attend every meeting. We
+In instances where a process is not already specified within this document,
+changes to the Steering Committee charter will be considered according to the
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Elections
+
+<!--
+
+- Consider explicitly mentioning SIG ContribEx's Elections subproject, once it
+exists.
+
+-->
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+In the instance that a general election is called as a result of the
+[dissolution](#dissolution) of the current committee, the following rules will
+apply:
+
+* members of the current committee will not be allowed to run in the new
+  election
+* a new committee will be bootstrapped with:
+  * the top four candidates serving a two-year term
+  * the next three candidates serving a one-year team
```

Comment:
```suggestion
  * the next three candidates serving a one-year team
* the Election Officers will be selected by the Elections Subproject and confirmed by the Contributor Experience Chairs.
```

We'll need a way to pick the EOs in the absence of a functioning SC.  My suggestion is above.  I'm not attached to having ContribEx pick the EOs, but we need to have someone other than the SC approve them (the process in a normal election).

Reply:
agree with dims - have contribex select the EO team as they do today and then the final approvers are the leads@ list. the only thing that needs to change is the approvers.

Reply:
Paris: you know as well as I do that folks don't respond on the leads@ list -- certainly we aren't going to get a majority of leads responding, if we get any at all.  So that would mean we'd need to do approval-by-lazy-consensus, which means that it would take a minimum of a week for the EOs to be approved.

Reply:
Again, the threat model here is "unable to reform the SC because holding a special election is too hard".   Let's not make that threat worse.

Reply:
There are multiple threat vectors here. You can't optimize for one only.

Perhaps we should go with, the election officers from the last election must run this one. And add that retroactively to the requirements for being an election officer, that you're potentially on the hook for running a special one. This isn't fair to the people who didn't sign up for that but hopefully they can forgive us.

Reply:
It's somewhat poetic for the security response committee to assist in rebooting the board. That's not what they signed up to do, but that likely makes them a good check on the system.

Reply:
I'm happy with any relatively small group of people who are known to be responsive.  SRC and/or COCC work for that.